src/e2e-test/java/io/cdap/plugin/utils/CdfPluginProperty.java

```diff
@@ -0,0 +1,33 @@
+package io.cdap.plugin.utils;
+
+/**
+ * Plugin property data-cy attributes of elements which will be used to locate element by xpath.
+ */
+public enum CdfPluginProperty {
```

Comment:
You can create a mapping b/w `projectId` and `CdfPluginProperty.PROJECT_ID`. Similarly for all properties. This way you need to maintain only one step definition for all.
For example,
```
private static final Map<String, CdfPluginProperty> CDF_PLUGIN_PROPERTY_MAPPING;
  static {
    CDF_PLUGIN_PROPERTY_MAPPING = new ImmutableMap.Builder<String, CdfPluginProperty>()
    .put("projectId", CdfPluginProperty.PROJECT_ID)
    .build();
    }
  @Nullable
  public static CdfPluginProperty fromPropertyString(String property) {
    return CDF_PLUGIN_PROPERTY_MAPPING.get(property);
  }
```

Reply:
Added mapping